inputs/make_metadata/02_filter_runinfo.R

```diff
@@ -0,0 +1,103 @@
+library(dplyr)
+library(readr)
+
+# this script reads in the ENA runinfo table, filters it back down to the
+# transcriptome samples we are investigating, and writes it back out.
+# It also investigates the size of the metadata table, and the identifiers
+# that should be used for this project.
+
+
+# read in and filter data -------------------------------------------------
+
+# these samples will need to be filtered out, as they were not included in the 
+# runinfo tables.
+jgi_sras_that_fail <- c("SRP235317", "SRP234344", "SRP234343", "SRP234342", 
+                        "SRP234341", "SRP234340", "SRP234339", 'SRP234338', 
+                        'SRP234337', 'SRP234336', 'SRP234335', 'SRP234334', 
+                        'SRP234333', 'SRP234332', 'SRP234331', 'SRP234330',
+                        'SRP234329', 'SRP234328', 'SRP234327', 'SRP234326', 
+                        'SRP234325', 'SRP234324', 'SRP234323', 'SRP234322', 
+                        'SRP234321', 'SRP234320', 'SRP234319', 'SRP234318',
+                        'SRP231538', 'SRP231537', 'SRP231536', 'SRP231535', 
+                        'SRP231534', 'SRP231533', 'SRP231129', 'SRP231128', 
+                        'SRP231127', 'SRP231126', 'SRP231120', 'SRP231119',
+                        "SRP231118", "SRP231117", 'SRP231035', 'SRP231034', 
+                        'SRP231033', 'SRP231032', 'SRP231031', 'SRP231030', 
+                        'SRP231029', 'SRP231028', 'SRP231027', 'SRP231026',
+                        'SRP231025', 'SRP231024', 'SRP231023', 'SRP231022', 
+                        'SRP231021', 'SRP231020', 'SRP231019', 'SRP231018', 
+                        'SRP231017', 'SRP231016', 'SRP231015', 'SRP231014',
+                        'SRP231013', 'SRP231012', 'SRP231011', 'SRP231010', 
+                        'SRP231009', "SRP230142", "SRP230135", "SRP230133")
```

Comment:
nitpick: Alternating between single and double quotes on the list is a bit unusual